### PR TITLE
Consistently adding periods in error messages raised in vernac

### PR DESCRIPTION
**Pull request by herbelin**
**Kind:** Global consistency in passing

### CODE REVIEW

File: vernac/record.ml [hunk 1]
```diff
@@ -477,7 +477,7 @@ let check_template ~template ~poly ~univs ~params { Data.id; rdata = { DataR.min
   match template with
   | Some template, _ ->
     (* templateness explicitly requested *)
-    if poly && template then user_err Pp.(strbrk "template and polymorphism not compatible");
+    if poly && template then user_err Pp.(strbrk "template and polymorphism not compatible.");
```
Comment (Alizter) [inline vernac/record.ml:480]:
```suggestion
    if poly && template then user_err Pp.(strbrk "Template and polymorphism not compatible.");
```

**Review by Alizter — APPROVED** [review]

File: vernac/record.ml (hunk 1 above)
Comment (herbelin) [reply vernac/record.ml:480]: I thought `template` had here to be considered a "keyword", thus keeping its case. So maybe "Attributes template and polymorphism are not compatible."?

Comment (Alizter) [reply vernac/record.ml:480]: That looks better, we could also put them in quotes.

Comment (herbelin) [reply vernac/record.ml]: Ok, put in quotes. Done

**Comment by Alizter** [comment]
merging tomorrow.

**Comment by Alizter** [comment]
@coqbot merge now

**Pull request merged**